README.md

```diff
@@ -117,10 +127,6 @@ See issue [#5120](https://github.com/OmniSharp/omnisharp-vscode/issues/5120) for
   * Update to latest .NET SDKs (PR: [omnisharp-roslyn##2378](https://github.com/OmniSharp/omnisharp-roslyn/pull/2378))
   * Remove MSBuild and Mono from release packages ([omnisharp-roslyn##2339](https://github.com/OmniSharp/omnisharp-roslyn/issues/2339), PR: [omnisharp-roslyn##2360](https://github.com/OmniSharp/omnisharp-roslyn/pull/2360))
 
-## What's new in 1.24.4
```

Comment:
?

Reply:
based on the instructions I have we only keep the last few releases in the readme (still available in the changelog).  So removing the oldest when I added a new one.

Reply:
Alright